app/components/agenda/agendaitem/agendaitem-decision.hbs

```diff
@@ -25,12 +25,11 @@
       {{/if}}
     </div>
     <div class="auk-u-mt-2">
+      {{! no decision result code in legacy data }}
```

Comment:
Deze commentaar klopt niet en mag gewoon weg, bij aanmaak van een nieuwe beslissing is er ook nog geen decision result code gekozen